client/components/Filter.jsx

```diff
@@ -1,86 +1,85 @@
-import React from 'react';
```

Comment:
Please revert changes back to original. (I have similar fixes on my current branch.)